logstash-integration-aws.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name            = "logstash-integration-aws"
-  s.version         = "7.0.1"
+  s.version         = "7.1.0"
```

Comment:
Nit: 

As we have a `VERSION` file, we should refer to it here too, to avoid a mismatch

Reply:
Good catch, thanks.
I think it is better to load the version from the `VERSION` file.